subjects/getascii/README.md

```diff
@@ -0,0 +1,34 @@
+## getascii
+
+### Instructions
+
+Write a program that takes a character as an argument, and displays the correspondent ASCII code followed by a newline.
+
+- If the input is more than one character , return a newline (`'\n'`).
```

Comment:
I would change this "..return a newline (`'\n'`)."  To --> "...return a newline `\n`.

Reply:
fixed